invenio_users_resources/services/generators.py

```diff
@@ -11,6 +11,8 @@
 """Permission generators for users and groups."""
 
 
+from flask import current_app
```

Comment:
what is probably missing is to the change the groups service config, and to change the `can_read` to take into account this new flag.

Reply:
Done!